export function captureException(err: Error, extra ?: any) {
  console.error (err, extra)
}
